package.json

```diff
@@ -4,20 +4,17 @@
   "version": "1.0.0",
   "scripts": {
     "frontend:watch": "cd frontend && npm start",
-    "backend:watch":  "cd backend && . env/bin/activate && python manage.py runserver",
-
+    "backend:watch": "cd backend && . env/bin/activate && python manage.py runserver",
     "//1": "These map 1-to-1 with entries in ./Profile",
     "proc:backend_runserver": "cd backend && ./setup_testing.sh runserver",
-    "proc:backend_test":      "cd backend && ./setup_testing.sh test",
-    "proc:frontend":          "cd frontend && npm start",
+    "proc:backend_test": "cd backend && ./setup_testing.sh test",
+    "proc:frontend": "cd frontend && npm start",
     "//2": "cypress_xxx use sleep 10 to allow DB setup & frontend server to spawn",
-    "proc:cypress_open":      "cd frontend && sleep 10 && npx cypress open",
-    "proc:cypress_run":       "cd frontend && sleep 10 && npx cypress run",
-
+    "proc:cypress_open": "cd frontend && sleep 10 && npx cypress open",
+    "proc:cypress_run": "cd frontend && sleep 10 && npx cypress run",
     "//3": "This runs the backend & the frontend dev-server",
     "test:frontend:open": "nf start backend_runserver,frontend,cypress_open",
-    "test:frontend:run":  "nf start backend_runserver,frontend,cypress_run",
-
```

Comment:
Nitpick but the alignment of this file doesn't need to be changed, it's more readable the way it was originally written. Aligned characters make it easy to spot the divergences across lines.